ogcapi-stable/ogcapi-foundation/src/main/java/de/ii/ogcapi/foundation/infra/rest/ExceptionMapper.java

```diff
@@ -152,16 +152,14 @@ private Response serverError(Throwable exception, ExceptionFormatExtension excep
 
   // TODO: detail message
   // TODO: maybe add a Retry-After HTTP header?
-  private Response serviceUnavailable(ExceptionFormatExtension exceptionFormat) {
+  private Response serviceUnavailable(ExceptionFormatExtension exceptionFormat, String message) {
     final Response.Status responseStatus = Status.SERVICE_UNAVAILABLE;
```

Comment:
It is good to have the option, but not every part of the code should be responsible for creating user-facing messages. There should be a generic default message for these cases.
```suggestion
    final Response.Status responseStatus = Status.SERVICE_UNAVAILABLE;
    final String msg = Objects.requireNonNullElse(message, "The requested resource is currently not available. Please try again later.");
```